sanowret-crystal.lic

```diff
@@ -11,38 +11,36 @@ class SanowretCrystal
         { name: 'run', regex: /run/i, description: 'Single run of the script', optional: true }
       ]
     ]
-    @invalid_rooms = []
     @worn_crystal = false
     args = parse_args(arg_definitions)
     settings = get_settings
-    @adjective = settings.sanowret_adjective
-    @no_use_scripts = settings.sanowret_no_use_scripts
-    check_crystal if args.run && !hiding? && !invisible?
+    @adjective = settings.sanowret_adjective || 'sanowret'
+    @no_use_scripts = settings.sanowret_no_use_scripts || []
```

Comment:
If this setting is in `base-empty.yaml`, we don't need the default value. If it isn't in there, it should be (and then we still don't need the default value).

Same with the other two settings.

The idea here is to avoid having every script try and provide default values - that's what the two `base` YAML files are for.

Reply:
Ahh, understood, thanks! -- I'll double-check the `base.yaml` and `base-empty.yaml` for those settings. It's finally clicked how those are used 😅

Reply:
Empty defaults added per https://github.com/rpherbig/dr-scripts/pull/4435/commits/b74a4c345c365b66c68db3127654a3945c50abf5